recipes/qvina/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "qvina" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hadim/qvina/archive/cmake.zip
+  sha256: b11a6e089dd00a528bc5ff6b83f79045311ec806c199963287bc0939166659f9
+
+build:
+  skip: True  # [win or osx]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.12
+    - make
+    - boost-cpp >=1.69.0
```

Comment:
boost should not be a build dependency, but only a host dep.  Packages link against boost (ie the host section) in the target architeture. The build architerure is only for compilers and other build tools